code/modules/cargo/expressconsole.dm

```diff
@@ -0,0 +1,112 @@
+/obj/machinery/computer/cargo/express
+	name = "express supply console"
+	desc = "This console allows the user to purchase a package for double the price,\
+		with 1/40th of the delivery time: made possible by NanoTrasen's new \"Drop Pod Railgun\".\
+		All sales are near instantaneous - please choose carefully"
+	icon_screen = "supply_express"
+	circuit = /obj/item/circuitboard/computer/cargo/express
+	blockade_warning = "Bluespace instability detected. Delivery impossible."
+	req_access = list(ACCESS_QM)
+	var/message
+	var/locked = TRUE
+	
+
+/obj/machinery/computer/cargo/express/attackby(obj/item/W, mob/user, params)
+	..()
+	if(istype(W, /obj/item/card/id) || istype(W, /obj/item/device/pda))
+		if(allowed(user))
+			locked = !locked
+			to_chat(user, "<span class='notice'>You [locked ? "lock" : "unlock"] the interface.</span>")
+
+/obj/machinery/computer/cargo/express/emag_act(mob/user)
+	if(emagged)
+		return
+	user.visible_message("<span class='warning'>[user] swipes a suspicious card through [src]!</span>",
+	"<span class='notice'>You change the routing protocols, allowing the Drop Pod to land anywhere on the station.</span>")
```

Comment:
Emags are generally not overtly obvious like this, use to_chat to user and maybe play the sparking sound if you want

Reply:
That's what's used for the supply and request console; why should the express one be any different?